packages/react-material-ui/src/index.ts

```diff
@@ -1,15 +1,8 @@
 export { default as Avatar } from './components/Avatar';
 export { default as Checkbox } from './components/Checkbox';
 
-import ContainerWithDrawer, {
-  ContainerWithDrawerProps,
-  MSWContainerWithDrawerProps,
-} from './components/ContainerWithDrawer';
-export {
-  ContainerWithDrawer,
-  ContainerWithDrawerProps,
-  MSWContainerWithDrawerProps,
-};
+import AppBar from './components/AppBar';
+export { AppBar };
```

Comment:
Can these import/export lines follow the same pattern as the ones above? Something like `export { default as AppBar } from './components/AppBar'` would be a good call